### PR TITLE
WMCO remove Azure support 4.10

### PR DESCRIPTION
**Pull request by mburke5678**
WMCO no longer supports Microsoft Azure in versions 5.0.0/4.10 and earlier.
https://github.com/openshift/windows-machine-config-operator/pull/1602

Current: [Understanding Windows container workloads](https://docs.openshift.com/container-platform/4.10/windows_containers/understanding-windows-container-workloads.html)
Preview: [Understanding Windows container workloads](http://file.rdu.redhat.com/mburke/winc-drop-azure-410/windows_containers/understanding-windows-container-workloads.html)
[Windows Containers release notes ](http://file.rdu.redhat.com/mburke/winc-drop-azure-410/windows_containers/windows-containers-release-notes-5-x.html)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: modules/wmco-prerequisites.adoc [hunk 1]
```diff
@@ -7,6 +7,12 @@
 
 The following information details the supported platform versions, Windows Server versions, and networking configurations for the Windows Machine Config Operator. See the vSphere documentation for any information that is relevant to only that platform.
 
+[IMPORTANT]
+====
+Because link:https://techcommunity.microsoft.com/t5/containers/important-update-deprecation-of-docker-virtual-machine-images/ba-p/3646272[Microsoft has stopped publishing Windows Server 2019 images with
+Docker], Red Hat no longer supports Windows Azure for WMCO releases older than version 6.0.0. For WMCO 5.y.z and earlier, Windows Server 2019 images must have Docker pre-installed. WMCO 6.0.0 and later uses containerd as the runtime. You can can either upgrade to {product-title} 4.11, which uses WMCO 6.0.0, or build a custom Windows image with Docker installed.
```
Comment (aravindhp) [inline modules/wmco-prerequisites.adoc:13]: I suggest removing ` or build a custom Windows image with Docker installed.` as that implies we support that and we don't.

**Comment by mtnbikenc** [comment]
LGTM

**Review by aravindhp — COMMENTED** [review]
LGTM

File: modules/wmco-prerequisites.adoc [hunk 2]
```diff
@@ -7,6 +7,12 @@
 
 The following information details the supported platform versions, Windows Server versions, and networking configurations for the Windows Machine Config Operator. See the vSphere documentation for any information that is relevant to only that platform.
 
+[IMPORTANT]
+====
+Because link:https://techcommunity.microsoft.com/t5/containers/important-update-deprecation-of-docker-virtual-machine-images/ba-p/3646272[Microsoft has stopped publishing Windows Server 2019 images with
+Docker], Red Hat no longer supports Windows Azure for WMCO releases older than version 6.0.0. For WMCO 5.y.z and earlier, Windows Server 2019 images must have Docker pre-installed. WMCO 6.0.0 and later uses containerd as the runtime. You can can either upgrade to {product-title} 4.11, which uses WMCO 6.0.0.
```
Comment (jfrancoa) [inline modules/wmco-prerequisites.adoc:13]: s/You can can either upgrade/You can upgrade/g
(I'm not native speaker, but isn't either used when you are about to give two options? ex: "You either chose going to the beah, or going to the sea". However, I only see one single alternative)

**Comment by xltian** [comment]
lgtm

File: modules/wmco-prerequisites.adoc [hunk 3]
```diff
@@ -7,6 +7,12 @@
 
 The following information details the supported platform versions, Windows Server versions, and networking configurations for the Windows Machine Config Operator. See the vSphere documentation for any information that is relevant to only that platform.
 
+[IMPORTANT]
+====
+Because link:https://techcommunity.microsoft.com/t5/containers/important-update-deprecation-of-docker-virtual-machine-images/ba-p/3646272[Microsoft has stopped publishing Windows Server 2019 images with
+Docker], Red Hat no longer supports Windows Azure for WMCO releases older than version 6.0.0. For WMCO 5.y.z and earlier, Windows Server 2019 images must have Docker pre-installed. WMCO 6.0.0 and later uses containerd as the runtime. You can upgrade to {product-title} 4.11, which uses WMCO 6.0.0.
```
Comment (kcarmichael08) [inline modules/wmco-prerequisites.adoc:13]:
```suggestion
Docker], Red Hat no longer supports Windows Azure for WMCO releases earlier than version 6.0.0. For WMCO 5.y.z and earlier, Windows Server 2019 images must have Docker pre-installed. WMCO 6.0.0 and later uses containerd as the runtime. You can upgrade to {product-title} 4.11, which uses WMCO 6.0.0.
```

File: windows_containers/windows-containers-release-notes-5-x.adoc [hunk 4]
```diff
@@ -15,6 +15,12 @@ The release notes for {productwinc} tracks the development of the WMCO, which pr
 
 Version 5.x of the WMCO is compatible only with {product-title} 4.10.
 
+[IMPORTANT]
+====
+Because link:https://techcommunity.microsoft.com/t5/containers/important-update-deprecation-of-docker-virtual-machine-images/ba-p/3646272[Microsoft has stopped publishing Windows Server 2019 images with
+Docker], Red Hat no longer supports Windows Azure for WMCO releases older than version 6.0.0. For WMCO 5.y.z and earlier, Windows Server 2019 images must have Docker pre-installed. WMCO 6.0.0 and later uses containerd as the runtime. You can upgrade to {product-title} 4.11, which uses WMCO 6.0.0.
```
Comment (kcarmichael08) [inline windows_containers/windows-containers-release-notes-5-x.adoc:21]:
```suggestion
Docker], Red Hat no longer supports Windows Azure for WMCO releases earlier than version 6.0.0. For WMCO 5.y.z and earlier, Windows Server 2019 images must have Docker pre-installed. WMCO 6.0.0 and later uses containerd as the runtime. You can upgrade to {product-title} 4.11, which uses WMCO 6.0.0.
```

File: modules/wmco-prerequisites.adoc (hunk 3 above)
Comment (kcarmichael08) [inline modules/wmco-prerequisites.adoc:13]:
```suggestion
Docker], Red Hat no longer supports Windows Azure for WMCO releases earlier than version 6.0.0. For WMCO 5.y.z and earlier, Windows Server 2019 images must have Docker pre-installed. WMCO 6.0.0 and later uses containerd as the runtime. You can upgrade to {product-title} 4.11, which uses WMCO 6.0.0.
```

**Review by kcarmichael08 — COMMENTED** [review]
One minor suggestion; otherwise, lgtm! I guess the build is failing because of github issues, so I am not sure if that requires trying to push it again. It looks like gh is having trouble getting vale to run. Also don't forget to check the QE box before submitting the merge request. I'll keep my fingers crossed that the build runs successfully!

**Pull request merged**